package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "2.0.36",
+  "version": "3.0.0",
```

Comment:
This shouldn't need a major version bump, since the application has not had a significant change.  The only major update is `jest-fetch-mock` and the changes do not impact the Psammead application. 

It should be just fine to have this PR have a patch update `2.0.37`.